.github/workflows/fixtures/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react';
+import { SafeAreaView, ScrollView, StatusBar } from 'react-native';
+import { AdMobBanner } from 'react-native-admob';
+
+const App = () => {
+  return (
+    <Fragment>
+      <StatusBar barStyle="dark-content" />
+      <SafeAreaView>
+        <ScrollView
+          contentInsetAdjustmentBehavior="automatic"
+          style={styles.scrollView}
```

Comment:
`styles.scrollView` is not defined because there is no StyleSheet

Reply:
Good catch!